dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/internal/domain/TracingFileStrategy.kt

```diff
@@ -0,0 +1,36 @@
+package com.datadog.android.tracing.internal.domain
+
+import android.content.Context
+import com.datadog.android.core.internal.domain.FilePersistenceStrategy
+import datadog.opentracing.DDSpan
+import java.io.File
+
+internal class TracingFileStrategy(
```

Comment:
It's kinda inconsistent with `com.datadog.android.log.internal.domain.LogFileStrategy`. Should be `com.datadog.android.trace.internal.domain.TraceFileStrategy` to keep the consistency, so both package should be `tracing` → `trace` and class `TracingFileStrategy` → `TraceFileStrategy`. *#naming* 🤷‍♂️🙂